frontend/src/PhoneNumberField.js

```diff
@@ -1,28 +1,24 @@
 import React from 'react'
-import { elementType, func, oneOfType, shape, string } from 'prop-types'
+import { string } from 'prop-types'
 import { useLingui } from '@lingui/react'
-import { t, Trans } from '@lingui/macro'
+import { Trans } from '@lingui/macro'
 import {
+  Box,
   FormControl,
   FormErrorMessage,
-  FormHelperText,
   FormLabel,
-  Icon,
-  Input,
-  InputGroup,
-  InputLeftElement,
 } from '@chakra-ui/core'
-import { useField } from 'formik'
+import { useField, useFormikContext } from 'formik'
 import WithPseudoBox from './withPseudoBox'
 import { fieldRequirements } from './fieldRequirements'
+import PhoneInput from 'react-phone-input-2'
```

Comment:
Can you take a stab at lazy loading this library? I remember it's pretty big (a massive improvement on libphonenumber-js but still big). Try this:

```js 
const PhoneInput = lazy(() => import('react-phone-input-2))
```
It's a default export, so I think [React.lazy](https://reactjs.org/docs/code-splitting.html#reactlazy) would work here. The result would be the JS bundle is split up more and chunks loaded as needed.
If it works, let's do that.